### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by DaveA50**

### CODE REVIEW

File: packaging/windows/lbry-win32-app/LBRYWin32App.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+import os
+import socket
+import sys
+import threading
+import webbrowser
+
+# from appdirs import user_data_dir
+from twisted.internet import reactor
+from twisted.web import server
+# from twisted.internet.cfreactor import install
```
Comment (jackrobison) [inline packaging/windows/lbry-win32-app/LBRYWin32App.py]: the os x imports can be deleted

File: packaging/windows/lbry-win32-app/LBRYWin32App.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    systray_thread = threading.Thread(target=LBRYApp)
+    systray_thread.start()
+
+    lbry = LBRYDaemonServer()
+    d = lbry.start()
+    d.addCallback(lambda _: webbrowser.open(UI_ADDRESS))
+    lbrynet_server = server.Site(lbry.root)
+    lbrynet_server.requestFactory = LBRYDaemonRequest
+    reactor.listenTCP(API_PORT, lbrynet_server, interface=API_INTERFACE)
+    # reactor.addSystemEventTrigger("after", "shutdown", AppHelper.stopEventLoop)
```
Comment (jackrobison) [inline packaging/windows/lbry-win32-app/LBRYWin32App.py]: without something to stop the app on reactor shutdown will the app close when you make a stop() api call?

**Comment by jackrobison** [comment]
screen shot 2016-08-23 at 12 14 06 am

File: packaging/windows/lbry-win32-app/LBRYWin32App.py (hunk 2 above)
Comment (DaveA50) [inline packaging/windows/lbry-win32-app/LBRYWin32App.py]: The program ends cleanly on api.stop() call now that the systray icon runs as a daemon thread

File: packaging/windows/lbry-win32-app/LBRYWin32App.py (hunk 1 above)
Comment (DaveA50) [inline packaging/windows/lbry-win32-app/LBRYWin32App.py]: Removed

**Pull request merged**